src/madek/api/resources/shared/media_resource_helper.clj

```diff
@@ -22,6 +22,7 @@
                                               (-> (sql/select :*)
                                                   (sql/from (keyword table-name))
                                                   (sql/where [:= :id (to-uuid id)])
+                                                  (sql/where [:raw "(deleted_at is null or now() < deleted_at)"])
```

Comment:
i would extract this condition as `(def soft-deleted-cond [:raw "(deleted_at is null or now() < deleted_at)"]))` and use it everywhere. the same thing for `not-soft-deleted` if necessary.